### DISCUSSION TITLE
connect to tenant database from system database

### DISCUSSION THREAD

**Discussion by mo3auya91** [hyn/multi-tenant#973] [Q&A]
Hello,
I am using hyn/multi-tenant,
this is what I've done
* in main domain (system database)
when a user registered in the main domain (system connection), he is redirected to create a project.
Then I created a subdomain and database for each project in the backend (behind the scenes).
and every subdomain works fine.

* in tenent domain (tenant database)
Every tenant database have a products table.

My question, how can the user (who created this project and I generated a subdomain and database for him) access his database (the tenant connection) from main website (system connection)?
It's kinda admin panel for the system user, he needs to manage his products in a tenant database?
How can I achieve that? 
Am I doing something wrong here?
Do I need to change the database structure to do that, if yes, what I need to change?

**Comment by mo3auya91** [comment]
@ArlonAntonius 
Could you help here please?

**Reply by ArlonAntonius** [reply]
Hey there,

Most of the time mentioning me doesn't actually help; I simply reply to Github Notifications whenever I have time and other team members are often a lot faster than me.

**Comment by ArlonAntonius** [comment]
To the general issue.

I would first try to map out how you want to structure things. Providing an "admin panel"-like experience is quite hard with this package as far as I remember. I think it comes down to you want to manually set the connection instead of completely setting the tenant.

Going to CC Bert (@bkintanar) here; he probably knows from the top of his head :)
